docs/single-page.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: "Single Page - FINOS AI Governance Framework"
+---
+
+<div class="container py-4">
+  <div class="d-flex justify-content-between align-items-center mb-4">
+    <h1>FINOS AI Governance Framework — Single Page</h1>
+    <div>
+      <a href="/" class="btn btn-outline-secondary btn-sm">Back</a>
+      <button class="btn btn-outline-primary btn-sm" onclick="window.print()">Print / Save as PDF</button>
+    </div>
+  </div>
+
+  <nav class="mb-4">
```

Comment:
when I print preview I cannot see the contents, I think this is because you are using a `nav` element which isn't visible in print view by default.